index.js

```diff
@@ -24,24 +61,45 @@ addTaskButton.addEventListener("click", () => {
     createdAt: new Date().toISOString(),
   });
 
+  allTasks = tasks;
+
   renderTasks();
 });
 
+searchButton.addEventListener("click", () => {
+  searchTitle(sreachInput.value);
+});
+
+taskSortSection.addEventListener("change", () => {
+  const sortType = taskSortSection.options[taskSortSection.selectedIndex].value;
+
+  sortTasks(sortType);
+});
+
+timeFilterSection.addEventListener("change", () => {
+  const selectedTime =
+    timeFilterSection.options[timeFilterSection.selectedIndex].value;
+
+  filterTasksByTime(selectedTime);
+});
+
 function renderTasks() {
   let tasksHtml = "";
+  let countTasks = 0;
```

Comment:
Redundant variable. Read `forEach` function and use `index` instead.

Reply:
such a great features it has ^^
I learned both array and index in `forEach`.
Hope I could use them in future.